### PR TITLE
bonus.com.tr

### PR DESCRIPTION
**Pull request by gork7777**
https://github.com/AdguardTeam/AdguardFilters/issues/76488

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -3148,6 +3148,7 @@ smashbros.com##.cookie-notes
 aptoide.com##.cookie-notification-holder
 calik.com,telekom.hu##.cookie-policy-container
 zebramo.com,modacruz.com##.cookie-policy-sticky
+bonus.com.tr###cookie-policy-sticky
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/cookies_specific.txt:3151]: After hiding an element, a white stripe remains

I would add these rules:
`bonus.com.tr#%#//scriptlet('set-cookie', 'cookie-agreement', '1')`
`bonus.com.tr#@#.cookie-content` - to cookies_whitelist

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_specific.txt:3151]: `bonus.com.tr###cookie-policy-sticky` it's ok for cb platforms. `bonus.com.tr#@#.cookie-content` I don't think this is necessary.

@gork7777 `cookies_specific.txt` add `bonus.com.tr#%#//scriptlet('set-cookie', 'cookie-agreement', '1')` under `JS rules`.
If you can't do that please let us know.

Comment (Alex-302) [reply AnnoyancesFilter/sections/cookies_specific.txt:3151]: If you hide it for content blocker, padding will not be removed, and message(leftover) cannot be closed by user.
image
Better just scriptlet or CSS.

Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_specific.txt:3151]: @Alex-302 isn't it removed after scrolling? Then I guess scriptlet / css are better here with exception, but without the basic rule from the pull request.

Comment (Alex-302) [reply AnnoyancesFilter/sections/cookies_specific.txt]: After scrolling:
image


Just scriptlet/CSS is required. Without exclusion, close button is visible.

Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_specific.txt]: Ok, thank you.

Comment (gork7777) [reply AnnoyancesFilter/sections/cookies_specific.txt]: Fixed and how did you find the 'cookie-agreement'? I don't know these scriptlet codes very well

**Review by zloyden — APPROVED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_specific.txt]: It's not me who found it. But you can see it after accepting cookies in devtools -> application -> cookies or look at their js code.

<details><summary>For instance:</summary>

image

</details><br/>

Comment (gork7777) [reply AnnoyancesFilter/sections/cookies_specific.txt]: Okay thanks for information!

Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_specific.txt]: You are welcome.

**Pull request merged**